addons/nuxeo-coldstorage/elements/nuxeo-move-content-to-coldstorage-button.js

```diff
@@ -0,0 +1,136 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/iron-icons/iron-icons.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '@nuxeo/nuxeo-elements/nuxeo-operation.js';
+import '@polymer/paper-button/paper-button.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/polymer/lib/elements/dom-if.js';
+import '@polymer/polymer/lib/elements/dom-repeat.js';
+import { FiltersBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-filters-behavior.js';
+import { FormatBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-format-behavior.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-icons.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-dialog.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-tooltip.js';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-action-button-styles.js';
+
+/**
+`nuxeo-move-content-to-coldstorage-button`
+@group Nuxeo UI
+@element nuxeo-move-content-to-coldstorage-button
+*/
+class MoveToColdStorage extends mixinBehaviors([FiltersBehavior, FormatBehavior], Nuxeo.Element) {
+  static get template() {
+    return html`
+      <style include="nuxeo-action-button-styles nuxeo-styles"></style>
+      <nuxeo-operation id="opMove" op="Document.MoveToColdStorage" input="[[document.uid]]"></nuxeo-operation>
+      <nuxeo-connection id="nxcon" user="{{currentUser}}"></nuxeo-connection>
+
+      <dom-if if="[[_isAvailable(document)]]">
```

Comment:
_isAvailable has two dependencies: document anda currentUser, so they should be reflected here as well:
```suggestion
      <dom-if if="[[_isAvailable(document, currentUser)]]">
```